includes/definitions/algcom-sm.yaml

```diff
@@ -0,0 +1,13 @@
+os: algcom-sm
+text: 'ALGCOM SM'
+type: power
+icon: algcom
+mib_dir: algcom
+over:
+    - { graph: device_current, text: Current }
+    - { graph: device_voltage, text: Voltage }
+    - { graph: device_temperature, text: Temperature }
+
+discovery:
+    - sysObjectID:
+          - .1.3.6.1.4.1.49136.100
```

Comment:
This conflicts with algcom-dc-ups. We don't have any other test data to show that 1.3.6.1.4.1.49136 is too broad for the dc-ups version.

Reply:
Yep, I'll have to reject it there.  One reason why this is a draft.

I also need to add the ability to walk multiple oids in yaml  Alternatively, I could use pre-fetch, but I think this is a better path forward.